### PR TITLE
Add link back to keepachangelog.com

### PR DESCRIPTION
**Pull request by amaltson**
Usually every time I'm adding entries to the changelog, I forget the format. I then open the browser and come back to the website to refresh my memory. This is fine, but contributors to the project might not know that the `CHANGELOG.md` follows a specific format, and add headings with different, non-conforming names.

Making it clear what the format is for the CHANGELOG should help level set any contributors.

I ❤️ this project so much! We use this format extensively.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # Change Log
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
+The format conforms to [keepachangelog.com](http://keepachangelog.com/).
```
Comment (MitMaro) [inline CHANGELOG.md]: I would have the link title be "Keep a Changelog", reflecting the title on the website. Also love the idea!

**Comment by amaltson** [comment]
@MitMaro done. Hope @olivierlacan likes it 😉

**Comment by olivierlacan** [comment]
Hmm. Not a bad idea. Sorry for being picky but I think we could make adherence to SemVer and 
this format one single sentence: 

```
# Change Log
All notable changes to this project will be documented in this file.

The format is based on [Keep a Changelog](http://keepachangelog.com/) 
and this project adheres to [Semantic Versioning](http://semver.org/).
```

An alternative for anti-SemVer folks would be:

```
# Change Log
All notable changes to this project will be documented in this file.

The format is based on [Keep a Changelog](http://keepachangelog.com/).
```

What do you think?

**Comment by MitMaro** [comment]
I use this on projects that cannot strictly follow semantic versioning. I like that one sentence version, and it would be easy enough for projects not following semantic versioning to switch to the shorter version.

**Comment by amaltson** [comment]
@olivierlacan I'm good either way honestly. Would just love a link back so people can find the format. Which do you prefer?

**Comment by amaltson** [comment]
Hey @olivierlacan, I took the SemVer one you said above. Good to 🚢 ? Thanks!

**Comment by olivierlacan** [comment]
Yep, looks good. :-)

**Pull request merged**